<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-19</y>
      <width>1091</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgba(0,0,0,230);
color: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kode_book">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: rgb(0,0,0);
border-bottom : 2px solid rgba(255,255,255);
background-color: transparent;
font-size: 18px;
font-weight: 500;
color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nickname">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
color : rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Masukkan kode booking</string>
    </property>
   </widget>
   <widget class="QLabel" name="hasil_email">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: 500;
font-size: 18px;
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hasil_tanggal">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: 500;
font-size: 18px;
color: white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hasil">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: 500;
font-size: 18px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 255, 255);
color: black;
font-size: 16px;
font-family: &quot;Poppins&quot;;
border-radius: 15px;
font-weight: 600;
</string>
    </property>
    <property name="text">
     <string>Kembali</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cek">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 255, 255);
color: black;
font-size: 16px;
font-family: &quot;Poppins&quot;;
border-radius: 15px;
font-weight: 600;
</string>
    </property>
    <property name="text">
     <string>Cek</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>650</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 14px;
color: yellow;
font-weight : 500;</string>
    </property>
    <property name="text">
     <string>Warning</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>680</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px;
color: white;
font-weight : 500;</string>
    </property>
    <property name="text">
     <string>* Datang sesuai jadwal yang ditentukan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>710</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px;
color: white;
font-weight : 500;</string>
    </property>
    <property name="text">
     <string>* Pastikan kode booking benar</string>
    </property>
   </widget>
   <widget class="QLabel" name="hasil_harga">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: 500;
font-size: 18px;
color: white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hasil_kode">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>480</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: 500;
font-size: 18px;
color: white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
